website/src/documentation/sdks/python-dependencies.md

```diff
@@ -26,460 +26,41 @@ behavior in the service. If you are using any of these packages in your code, be
 aware that some libraries are not forward-compatible and you may need to pin to
 the listed versions that will be in scope during execution.
 
-<p>To see the compile and runtime dependencies for your Beam SDK version, expand
-the relevant section below.</p>
+Dependencies for your Beam SDK version dependencies are listed in `setup.py` in the Beam repository. To view them, perform the following steps:
```

Comment:
'dependencies' is repeated twice here as well. Let's shorten this to `Dependencies for your Beam SDK are listed in...`